example/transactions/Basic_38_TxTypeCancel.js

```diff
@@ -0,0 +1,63 @@
+const ethers = require("ethers");
+const { KlaytnWallet } = require("../../dist/src/ethers"); // require("@klaytn/sdk-ethers");
+
+const fs = require('fs')
+const sender_priv = fs.readFileSync('./example/privateKey', 'utf8') 
+
+const sender = '0x3208ca99480f82bfe240ca6bc06110cd12bb6366' 
+const reciever = '0xc40b6909eb7085590e1c26cb3becc25368e249e9'  
+
+//
+// TxTypeCancel
+// https://docs.klaytn.foundation/content/klaytn/design/transactions/basic#txtypecancel
+// 
+//   type: Must be 0x38,
+//
+// 1) send ValueTransfer tx with the original nonce+1  
```

Comment:
please call it 'next nonce'. e.g. in Klaytnscope https://scope.klaytn.com/account/0x186de0382923086f73367bab16af09aeda4e54bf?tabId=txList

Reply:
Updated the term.  (original nonce -> next nonce)

https://github.com/blukat29/klaytn-sdk-design/pull/2/commits/bf4a396c034fbe6e7e6a4a83fa3c266cb89d876b